gbucket-snippets/snippets_setup.R

```diff
@@ -0,0 +1,2 @@
+library(tidyverse)  # Data wrangling packages.
```

Comment:
I'd probably use `gcs-snippets` as the directory name, as `gbucket` isn't standard.

Reply:
@rfrancis1 I'm planning to send a rename pull request for `snippets` -> `sql-snippets`. I'd be happy to `gbucket-snippets` -> `gcs-snippets` in that same pull request too.